pages/api/hello.js

```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
wasn't this already in the original template? I'm a bit confused about all the data in this pull request. it's a lot! :)